lib/mongo/Mutator.js

```diff
@@ -152,7 +152,7 @@ export default class Mutator {
                 Originals,
                 selector,
                 modifier,
-                config,
+                _.extend({}, {_returnObject: false}, config),
```

Comment:
I think that `Object.assign` will suffice here:
https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_extend